src/encoded/types/experiment.py

```diff
@@ -1014,12 +1015,25 @@ def experiment_categorizer(self, request, experiment_type, biosample, imaging_pa
         if imaging_paths:
             path_targets = []
             for pathobj in imaging_paths:
-                path = request.embed('/', pathobj['path'], '@@object')
+                path = get_item_or_none(request, pathobj['path'], 'imaging_path')
                 for target in path.get('target', []):
-                    summ = request.embed('/', target, '@@object')['display_title']
+                    summ = get_item_or_none(request, target, 'bio_feature')['display_title']
                     path_targets.append(summ)
             if path_targets:
-                value = ', '.join(list(set(path_targets)))
+                value = []
+                sum_targets = {}
+                for target in path_targets:
+                    # check if target starts with numbers, e.g. '50 TADs', '40 TADs'
+                    # sum them if there are more: '90 TADs'
+                    split_target = re.split(r'(^[0-9]+)([^0-9].*)', target)
```

Comment:
Are you sure about this re.split?  It looks a bit tricky.  I guess it works as expected in python 3.6 but I think the second paren capture statement will return an empty string in v3.7 and above - so it may possibly not work exactly as expected in the future ?  Also is the not digit at the start of the second capture required?  Maybe a greedy ? at the end of the first capture would be safer?  And it seems we will need to take care to make sure that targets use the format expected - can we document this somewhere if that is the case?

Reply:
Right, the second capture group was not necessary. It should be fixed now.
The use case here is when target(s) of ImagingPath(s) start with numbers, and the rest of the text is identical. Very likely, this is only obtained by using a `preferred_label` for the BioFeature, which is already an advanced use case (typically, wranglers take care of it). I agree that documenting is necessary, although nothing happens in case the text is not formatted properly: targets will just remain split, as it has been until now.
The test should help understanding how it works, and now I also added docstrings. Any suggestion on how to document better how to use this?